stdlib/public/TensorFlow/Utilities.swift

```diff
@@ -290,7 +290,7 @@ public func _hostOp<Scalar>(_ x: Tensor<Scalar>) {
 }
 
 @inline(never)
-public func _hostOp<Scalar>(_ x: TensorHandle<Scalar>) {
+public func _hostOp<Scalar : AccelerableByTensorFlow>(_ x: TensorHandle<Scalar>) {
```

Comment:
Fit this line within 80 columns. In this case, you can put a `where Scalar : AccelerableByTensorFlow` clause at the end.